### PR TITLE
enhancement: link to nft details from nft activity popup

### LINKED ISSUE #5450: [Task]: Add link from NftActivityDetails to CollectiblesDetailsView
### Task description

When receiving / minting NFTs, users might want to easily access the details view of the specific NFT without needing to traverse through the CollectiblesGallery and find the specific NFT. Adding a direct link to the DetailsView from the ActivityDetails component would make this much easier for the user.

### Requirements

Make the NFT name in NftActivityDetails a link that routes to the specified NFT's details view.

### Creation checklist

- [X] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [X] I have linked the correct milestone and/or project

### PR DESCRIPTION
**Pull request by kraftjs**
## Summary

> Please summarize your changes, describing __what__ they are and __why__ they were made.

When receiving / minting NFTs, users might want to easily access the details view of the specific NFT without needing to traverse through the CollectiblesGallery and find the specific NFT. Adding a direct link to the DetailsView from the ActivityDetails component would make this much easier for the user.

## Changelog

```
- Add button to NftActivityDetails that is shown when nft is in selectedAccountNfts
- Fix placeholder borders showing behind loaded image
- Fix viewOnExplorer button taking up full parent width
```

## Relevant Issues

> Please list any related issues using [development keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

Closes #5450 

### Platforms

> Please select any platforms where your changes have been tested.

- __Desktop__
  - [x] MacOS
  - [ ] Linux
  - [ ] Windows
- __Mobile__
  - [ ] iOS
  - [ ] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to __test and verify__ that your changes work as intended.

...

## Checklist

> Please tick all of the following boxes that are relevant to your changes.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or modified tests that prove my changes work as intended
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have verified that my latest changes pass CI workflows for testing and linting
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/shared/components/molecules/NftActivityDetails.svelte [hunk 1]
```diff
@@ -19,16 +21,36 @@
...
 
 <nft-transaction-details class="w-full space-y-6 flex flex-auto flex-col flex-shrink-0">
     <main-content class="flex flex-auto w-full flex-col items-center justify-center space-y-3 overflow-hidden">
-        <nft-summary class="flex w-full items-center justify-center w-full space-x-2">
-            <NftImageOrIconBox nftId={activity.nftId} size="small" />
-            <Text type={TextType.h3} fontWeight={FontWeight.semibold} classes="whitespace-pre truncate">
-                {nft?.name}
-            </Text>
-        </nft-summary>
+        {#if $selectedAccountNfts.some((nft) => nft.id === activity.nftId)}
```
Comment (MarkNerdi) [inline packages/shared/components/molecules/NftActivityDetails.svelte:35]: you could just disable the `button` instead of copying the component

Comment (kraftjs) [reply packages/shared/components/molecules/NftActivityDetails.svelte:35]: I was still getting the button cursor when hovering over it and wasn't sure the best way to disable it. Guess I could give it a class that toggles whether the button is active.

Comment (kraftjs) [reply packages/shared/components/molecules/NftActivityDetails.svelte]: Went ahead isolated it all to a single button and added a conditional class to change the cursor styling

**Review by MarkNerdi — APPROVED** [review]
LGTM, gj!

**Pull request merged**